src/Objects/Preferences.vala

```diff
@@ -0,0 +1,54 @@
+public class Preferences : Object {
+    private Settings settings;
+
+    public string editor_font { get; set; default = ""; }
+    public string editor_scheme { get; set; default = ""; }
+    public string render_stylesheet { get; set; default = ""; }
+    public bool prefer_dark_theme { get; set; default = false; }
+    public int autosave_interval { get; set; default = 0; }
+
+    public void load () {
+        string schema_id = "org.markmywords";
+        string settings_dir = "../schemas";
+        SettingsSchema schema = null;
+        try { // Get schema from build dir if available
+            var sss = new SettingsSchemaSource.from_directory (settings_dir, null, false);
```

Comment:
this is pretty cool, no need to do make install for every change to the schema :+1: